django/db/backends/oracle/schema.py

```diff
@@ -200,7 +200,7 @@ def _generate_temp_name(self, for_name):
     def prepare_default(self, value):
         # Replace % with %% as %-formatting is applied in
         # FormatStylePlaceholderCursor._fix_for_params().
```

Comment:
Assuming Oracle tests pass I'll push another comment that removes this value. Otherwise I'll have to dust out my local Oracle setup 😅

Reply:
@charettes so following our conversation about the Oracle tests passing, the comment above still needs removing? Thanks!

Reply:
It does yes, I spotted it mid-Oracle test run so I wanted to avoid pushing a change that would require me to start over. I'll push a commit to this effect now that we know tests are passing.